### PR TITLE
Improve `formatRadicleId` util fn

### PR DESCRIPTION
**Pull request by sebastinez**
- Improves detection of Heartwood Radicle ids.
- Instead of throwing an error if the input isn't a Radicle id we return the original input and display a dev console warning

Signed-off-by: Sebastian Martinez <me@sebastinez.dev>

### CODE REVIEW

File: tests/unit/utils.test.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ describe("Format functions", () => {
 
   test("formatRadicleId", () => {
     if (process.env.HEARTWOOD) {
-      expect(utils.formatRadicleId("rad:zKtT7DmF9H34KkvcKj9PHW19WzjT")).toEqual(
-        "rad:zKtT7D…19WzjT",
+      expect(utils.formatRadicleId("zKtT7DmF9H34KkvcKj9PHW19WzjT")).toEqual(
+        "zKtT7D…19WzjT",
```
Comment (cloudhead) [inline tests/unit/utils.test.ts:24]: Hmm I'm not sure why we're changing this. The above is a valid radicle id in urn format. We also need to specify whether we're talking about repository ids or public keys..

File: src/lib/utils.ts [hunk 2]
```diff
@@ -163,7 +166,13 @@ export const formatTimestamp = (
 // Check whether the input is a Radicle ID.
```
Comment (cloudhead) [inline src/lib/utils.ts:166]: There's no such thing as a "radicle id" anymore in heartwood. There are node ids (or NIDs), which are public keys, and repository ids (or RIDs) which are Oids.

Comment (cloudhead) [reply src/lib/utils.ts:166]: The former is 32 bytes long, and the latter is 20 bytes long. Both use a multi-base encoding using the bitcoin base58 encoding, though RIDs are often displayed as URNs, with the `rad:` prefix.

File: tests/unit/utils.test.ts (hunk 1 above)
Comment (sebastinez) [reply tests/unit/utils.test.ts:24]: Interesting, yeah I think we need some design spec on all the identifiers.

Current state AFAIK
```
did:key:<public key>
Returned e.g.from delegate listings, seems more full format

rad:<public key>
Is this for projects only? Why do users don't have rad: prefixes?

<public key>
Defined as PublicKey or ActorId in httpd
```

I'm also ok with maybe a more general format function that can hande different identifier formats

File: src/lib/utils.ts (hunk 2 above)
Comment (sebastinez) [reply src/lib/utils.ts:166]: Ok this solves my other response wrt to a spec. So we need to two format functions, one for Oids with or without the prefix and one for public keys

**Comment by sebastinez** [comment]
After some brainstorming, I'll close this PR for a more fleshed out one, that cleans up after all the different ids we have floating around

**Pull request closed without merging**